kentik_api_library/kentik_api/api_connection/api_connector.py

```diff
@@ -58,16 +60,16 @@ def send(self, api_call: APICall, payload: Optional[Dict[str, Any]] = None) -> A
         return APICallResponse(response.status_code, response.text)
 
     def _do_request(self, api_call: APICall, payload: Optional[Dict[str, Any]] = None) -> Response:
-
         url = self._get_api_query_url(api_call.url_path)
+        headers = {"User-Agent": self._client_version}
         if api_call.method == APICallMethods.GET:
-            response = self._session.get(url, params=payload, timeout=self._timeout)
+            response = self._session.get(url, params=payload, headers=headers, timeout=self._timeout)
```

Comment:
Set `self._session.headers` in the constructor as opposed to passing it in every call.

Reply:
Updating